kong/client.go

```diff
@@ -365,3 +365,7 @@ func (c *Client) RootJSON(ctx context.Context) ([]byte, error) {
 
 	return body, nil
 }
+
+func (c *Client) BaseRootURL() string {
```

Comment:
WDYT about naming it `DefaultRootURL` to make it clear this is a getter for `c. defaultRootURL`?

Reply:
I somehow don't like the `default` naming in here 🤔 Reading this as a user I'd get a feeling that I'm getting a default value that could be used if nothing else was specified.

Reply:
I'd say the `defaultRootURL` field naming is misleading. It should be named IMO `rootURL` instead as it's not always falling back to the default one.

Reply:
Vey much so. Let it be that we fix this as we go. PTAL

Reply:
BTW the part calculating it in the constructor could be extracted to a separate function/method to make it more obvious. 🤔